packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -178,6 +181,7 @@ const BpkChipGroup = ({
           role={ariaMultiselectable ? 'group' : 'radiogroup'}
         >
           {ariaLabel && <legend className='visually-hidden'>{ariaLabel}</legend>}
+          {label && <BpkText textStyle={TEXT_STYLES.footnote} aria-hidden>{label}</BpkText>}
```

Comment:
Conditionally show the label
- can be aria-hidden because we use the legend for the non-visual representation
- defaulting to `footnote` size here - we could make it overridable but erring on keeping simple given we have no use case for that yet